packages/@react-aria/landmark/test/useLandmark.test.tsx

```diff
@@ -80,12 +80,11 @@ describe('LandmarkManager', function () {
   });
 
   afterAll(function () {
-    offsetWidth.mockReset();
+    offsetWidth.mockRestore();
```

Comment:
Remnant from my old changes when I had mocked the console.warn, kept the mockRestore change since it feels more accurate than mockReset from what I saw from the docs: https://jestjs.io/docs/mock-function-api#mockfnmockrestore.